product-backend/scripts/create_tables.sql

```diff
@@ -20,4 +20,14 @@ CREATE TABLE IF NOT EXISTS Specifications (
     summary VARCHAR(500),
     sharepoint_link VARCHAR(500),
     FOREIGN KEY (role_id) REFERENCES JobRoles(job_role_id)
-)
+    );
+
+  -- US013 add new capability
+    CREATE TABLE IF NOT EXISTS  `Capability` (
+        capability_id INT PRIMARY KEY AUTO_INCREMENT,
+        capability_name VARCHAR(80),
+        lead_name VARCHAR(80),
+        lead_photo MEDIUMBLOB,
+        lead_message VARCHAR(500)
+    );
+
```

Comment:
Remember to alter Band table (it's in band.sql) with FK to Capability and to keep the order (first create both tables, then add FK)